lib/db.js

```diff
@@ -758,13 +761,13 @@ module.exports = (
                       return true
                     }
 
-                    if (device.sessionToken === sessionTokenId) {
+                    if (device.sessionToken === sessionTokenId) { // TODO: Add refreshtoken support here?
```

Comment:
Catch up sometime tomorrow to discuss?

I think it will be OK to add a matching `if (device.refreshToken === refreshTokenId)` clause, but we should talk through the details.

Reply:
sounds good!